cli/src/program.rs

```diff
@@ -1537,7 +1607,7 @@ fn process_program_upgrade(
 
     let blockhash = blockhash_query.get_blockhash(&rpc_client, config.commitment)?;
     let message = Message::new_with_blockhash(
-        &[bpf_loader_upgradeable::upgrade(
```

Comment:
in the future, try to avoid renames like this in changes intended for backport. save cosmetics are refactors for master

Reply:
In this case it was not cosmetics. Just using `solana_program::bpf_loader_upgradeable` was not viable any longer because only `solana_loader_v3_interface` contains the added interface. I could have used both at different parts in the code, but I deemed that even worse.